server/app/views/admin/reporting/AdminReportingShowView.java

```diff
@@ -64,8 +64,9 @@ public Content render(
       CiviFormProfile profile,
       String programSlug,
       String programName,
+      String localizedProgramName,
       ReportingService.MonthlyStats monthlyStats) {
-    var title = String.format("%s reporting", programName);
+    var title = String.format("%s reporting", localizedProgramName);
```

Comment:
question (blocking): Does "reporting" need to be localized?

Reply:
Since only Admins view it, we can leave it as it. I am not sure how much we localize in Admin path